integration/travis/run_integration.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-set -ev
```

Comment:
It would be better to `timeout ... || curl_error=true` and then handle that specific error instead of turning of `set -e`.  `set -e` helps avoid a lot of issues with bash scripts IMO.